packages/shadowfinder/PKGBUILD

```
@@ -0,0 +1,51 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=shadowfinder
+_pkgname=ShadowFinder
+pkgver=0.5.0.r0.g578d5ac
+pkgrel=1
+pkgdesc='Find possible locations of shadows around the world.'
+arch=('any')
+url='https://github.com/bellingcat/ShadowFinder'
+license=('MIT')
+depends=('python' 'python-basemap' 'python-matplotlib' 'python-fire' 'python-numpy' 'python-pandas' 'python-pytz' 'python-suncalc' 'python-timezonefinder')
+makedepends=('git' 'python-build' 'python-pip')
+options=(!emptydirs)
+source=("git+https://github.com/bellingcat/$_pkgname.git")
```

```suggestion
source=("git+https://github.com/bellingcat/$pkgname.git")
```

Github namespaces are case insensitive, so you can use the lowercase vversion of the name for the URL and then to `cd` into the cloned folder.